CONTRIBUTING.md

```diff
@@ -215,16 +215,19 @@ flutter test test/ --update-goldens
 
 ## Manual Testing
 
-To run master version of DevTools with all experimental features enabled:
+If you do not have environment configured, but want to try newest DevTools with all experimental features enabled:
 
-1. Start DevTools
+1. Configure Dart or Flutter.
```

Comment:
if we are requiring them to configure flutter, then we don't need to say "If you do not have your environment configured". Can we change the above sentence to "If want to manually run DevTools with all experimental features enabled:"

Reply:
Also link to flutter getting started instructions